<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1480</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="tree_directory_view">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>231</width>
      <height>491</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="lbl_cur_direct">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>current directory:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_file_table">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>651</width>
      <height>491</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>file name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>modified time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>file type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>file size</string>
     </property>
    </column>
   </widget>
   <widget class="QPlainTextEdit" name="edt_textbox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>140</y>
      <width>461</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_file_name">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>90</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>file name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_overview">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>tree directory overview:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>661</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Operation System Assignment 3: File Management Demo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_format">
    <property name="geometry">
     <rect>
      <x>1089</x>
      <y>660</y>
      <width>342</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>format</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new_file">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>660</y>
      <width>342</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>new file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new_directory">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>660</y>
      <width>342</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>new directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save_exit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>660</y>
      <width>342</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>save and exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>967</x>
      <y>580</y>
      <width>226</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>580</y>
      <width>226</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
